doc/developers/advanced_installation.rst

```diff
@@ -28,6 +28,7 @@ Installing a nightly build is the quickest way to:
 
 .. prompt:: bash $
 
+  pip uninstall scikit-learn --yes
   pip install --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
```

Comment:
```suggestion
  pip install -U --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
```

Would this also work? It is what I used last time when I realised that i had spent 5min questioning my sanity because the change that I knew was in the nightly didn't appear.

Reply:
damn you were faster :)

Reply:
Right. In that case the appropriate option should be `--force-reinstall`